.releaserc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "branches": ["main", "develop"],
+  "plugins": [
+    ["@semantic-release/commit-analyzer", {
```

Comment:
Need to use the conventional commits plugin as well.

Reply:
Should be resolved.